tests/no-bad-latin.py

```diff
@@ -89,29 +104,35 @@ def read_and_check_files(files):
     return failing_files
 
 
-def construct_error_message(files_dict):
-    """Function to construct an error message pointing out where bad latin
-	phrases appear in lines of text
+def get_all_files(directory=os.path.join(ABSOLUTE_HERE,"book", "content")):
```

Comment:
If this function is called without an argument the content path might trigger an error in this case too.